tasks/gene_typing/variant_detection/task_snippy_variants.wdl

```diff
@@ -85,23 +85,55 @@ task snippy_variants {
     reference_length_passed_depth=$(cat "~{samplename}/~{samplename}_depth_~{min_coverage}.tsv" | wc -l)
     echo $reference_length_passed_depth | tee REFERENCE_LENGTH_PASSED_DEPTH
 
-    # check if reference_length is equal to 0, if so, output a warning
     if [ "$reference_length" -eq 0 ]; then
-      echo "Could not compute percent reference coverage: reference length is 0" > PERCENT_REF_COVERAGE
+      echo "0" > PERCENT_REF_COVERAGE
```

Comment:
I would hesitate to update this to be "0" because "0" means something different than "we can't calculate this"
@AndrewLangvt do you have an opinion? It has been "Could not compute" in several releases now, I think.

Reply:
@sage-wright @fraser-combe, after chatting with both of you & looking into snippy further, I think we shouldn't be dropping "0" as an output metric for this, nor for the PERCENT_READS_ALIGNED. 

PERCENT_REF_COVERAGE
The -a flag used in samtools depth results in all positions being output, even if 0 coverage. As such, this is accurately assessing the actual length provided of the reference. We wouldn't want to assign 0 to percent ref coverage, as reference_length==0 would mean that there actually was a 0-length reference provided. 

PERCENT_READS_ALIGNED
samtools view -c -F 4 will return all aligned reads (-F 4 excludes unmapped reads). This becomes our numerator. We again use samtools view -c (no -F 4 this time) to capture a numerator. In this instance, we're looking to see how many total alignments (mapped & unmapped) exist. We should expect this to not be zero, unless there was some fundamental issue with the alignment to begin with. Similar to above, we wouldn't want to assign a 0 here either, as if there truly were no alignments, we would still expect there to be unmapped reads identified in the samtools view -c command (our denominator). As such, when we truly don't have any aligned reads, but snippy processed successfully, our numerator will be 0, and the denominator will not. If we run into an instance where the denominator is 0 (i.e. no alignments in the BAM - mapped or unmapped), we can assume something has gone awry with the algorithm execution, itself. 

Quite a rabbit hole I went down, but all that to say - I think a string depicting "could not compute" or even "NA" would be more acceptable. The longer "could not compute..." string would potentially impact readability of the Terra table, but also would allow for ease of troubleshooting, vs if it was just "NA."

Reply:
I vote to maintain the current language -- it's more wordy & clear, but the columns are expandable.

Reply:
I can revert back to original wording then